man/newgidmap.1.xml

```diff
@@ -90,9 +90,9 @@
       The <command>newgidmap</command> sets <filename>/proc/[pid]/gid_map</filename> based on its
       command line arguments and the gids allowed. The subid delegation can come either from files
       (<filename>/etc/subgid</filename>) or from the configured NSS subid module. Only one of them
-      can be chosen at a time. So, for example, if the subid source is configured as NSS and
-      <command>groupadd</command> is executed, then the command will fail and the entry will not be
-      created in <filename>/etc/subgid</filename>.
+      can be chosen at a time. So, for example, if any subid source except files is configured and
```

Comment:
"Subgid delegation can either be managed via /etc/subgid or through the configured NSS subid module. These options are mutually exclusive.
Note, that groupadd will only create entries in /etc/subgid if subid delegation is managed via subid files."

Seems a bit clearer to me.

Reply:
Yes, thanks for the feedback. I'll update it accordingly.